<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConcordanceTab</class>
 <widget class="QWidget" name="ConcordanceTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Concordanceur</string>
  </property>
  <property name="locale">
   <locale language="French" country="France"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="concordanceSourceGroupBox">
     <property name="title">
      <string>Recherche d'un mot</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QLabel" name="concordanceSourceWordsBeforeLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Nombre de mots avant</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QSpinBox" name="concordanceSourceWordsAfterSpinBox">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="value">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="concordanceSourceWordsAfterLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Nombre de mots après</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="concordanceSourceWordsBeforeSpinBox">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="value">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QLineEdit" name="concordanceSourceLineEdit">
        <property name="placeholderText">
         <string>Votre mot</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="concordanceSourceSelectPushButton">
        <property name="text">
         <string>Sélectionner</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="4">
       <widget class="QGroupBox" name="concordanceSourceSearchTypesGroupBox">
        <property name="title">
         <string>Types de recherche</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCheckBox" name="concordanceSourceBasicSearchTypeCheckBox">
           <property name="text">
            <string>Basique</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="concordanceSourceLemmeSearchTypeCheckBox">
           <property name="text">
            <string>Basée lemme</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="concordanceSourceRootSearchTypeCheckBox">
           <property name="text">
            <string>Basée racine</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="concordanceStrictSearchCheckBox">
           <property name="text">
            <string>Stricte</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="concordanceSearchPushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Rechercher</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="concordanceResultsGroupBox">
     <property name="title">
      <string>Résultats</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTextEdit" name="concordanceResultsTextEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="concordanceResultsSavePushButton">
          <property name="text">
           <string>Enregistrer</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
